### DISCUSSION TITLE
Foreign key constraint failed on the field: `Factura_producto_id_fkey (index)`

### DISCUSSION THREAD

**Discussion by gmcamposano** [prisma/prisma#16086] [Q&A] [answered]
I am getting Foreign key constraint field error and I can't seem to pinpoint where the error is located. Can anyone guide me on how to solve this / why is this happening?
```
model Producto {
  id         String    @id @default(uuid())
  codigo     Int       @unique
  nombre     String
  precio     Decimal
  tienda     String
  created_at DateTime  @default(now())
  updated_at DateTime  @updatedAt
  Factura    Factura[]
}

model Factura {
  id                String   @id @default(uuid())
  facturado_por     String
  forma_de_pago     String
  tipo_de_factura   String
  numero_de_factura Int      @unique @default(autoincrement())
  rnc_empresa       Int?
  nombre_empresa    String?
  cantidad          Int
  total             Int
  Producto          Producto @relation(fields: [producto_id], references: [id])
  producto_id       String
}
```

**Comment by nurul3101** [comment, marked as answer]
Hey @gmcamposano 👋 
You get this error while invoking a Prisma Query, right?
Can you please share the query, so that I can investigate?

**Reply by gmcamposano** [reply]
Hello @nurul3101 
I tried deleting this post, but did not find how to do it!

Problem:  I was trying to add an incorrect reference to a table. From the example above => producto.id did not match producto_id, which I believe result in this error. I made sure it was the same and things started to work correctly. Sorry for this newbie question.

**Reply by nurul3101** [reply]
No worries at all 👍 
I am glad to hear that you were able to solve the issue! 🚀 
Please feel free to open a new discussion in case you face any other issues.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
